.ruby-version

```diff
@@ -1 +1 @@
-2.3.3
+2.6.0
```

Comment:
@masukomi This version seems to only be in preview for rbenv ? Would 2.5.3 be new enough ? ( I wonder if the 2.6 preview has additional debug code, which is what causes the slowness of the tests.